### PR TITLE
fix: upgrade storybook

### PR DESCRIPTION
**Pull request by AnnaRybkina**
For testing:
1. Run `pnpm install`
2. Try to run storybook `pnpm run dev` without any changes in package.json
3. Observe the behavior in the component on the Storybook pages. Should work as expected

Feature|Reaction
---|---
Dark mode support|

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,11 +54,12 @@
...
     "@semantic-release/git": "^10.0.1",
-    "@storybook/addon-actions": "^6.5.0-beta.0",
+    "@storybook/addon-actions": "^7.0.7",
+    "@storybook/addon-mdx-gfm": "^7.0.7",
     "@storybook/addon-postcss": "^2.0.0",
-    "@storybook/builder-webpack5": "^6.5.0-beta.0",
-    "@storybook/manager-webpack5": "^6.5.0-beta.0",
-    "@storybook/react": "^6.5.3",
+    "@storybook/cli": "^7.0.7",
+    "@storybook/react": "^7.0.7",
```
Comment (BalbinaK) [inline package.json:61]: Is this still used somewhere?

**Review by BalbinaK — APPROVED** [review]
You're amazing! It works like a charm!

File: package.json (hunk 1 above)
Comment (AnnaRybkina) [reply package.json]: Clean up a little bit https://github.com/warp-ds/react/pull/27/commits/cc7367baf84f82a7f4bfe679cb034ada2611c3e6 🙌  What do you think?

**Review by BalbinaK — APPROVED** [review]
Can't wait to use this!

**Pull request merged**